src/DssSpell.t.sol

```diff
@@ -258,7 +276,8 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(MedianAbstract(TOKENUSD_MED).bud(SET_TOKEN), 1);
     }
 
-    function testPSMs() public { // make private to disable
+    // leave puclic for now as this is acting like a config tests
```

Comment:
```suggestion
    // leave public for now as this is acting like a config tests
```

should we move its substance to base like the other "core" tests?

Reply:
oh damnit, this typo will be on mainnet too.